.github/workflows/yum-arm.yml

```diff
@@ -51,6 +52,11 @@ jobs:
         with:
           name: packages-${{ matrix.rake-job }}-aarch64
           path: fluent-package/yum/repositories
+      - name: Upload fluent-release rpm
+        uses: actions/upload-artifact@master
```

Comment:
`actions/upload-artifact@v4` is used in `.github/workflows/yum.yml`

What difference are `@v4` and `@master`?